### PR TITLE
Fixes for World Walkers

### PR DESCRIPTION
**Pull request by Mjb141**
- Cloned Grimgor's ability for Wulfrik. Needs flavour changes and then DB files need to remove Grimgors ability from him.
- Daemonic units update also included here in `building_units_allowed_tables`. Also needs https://github.com/msolefonte/vco/pull/212 merging.

### CODE REVIEW

**Comment by Mjb141** [comment]
1) https://cdn.discordapp.com/attachments/1021682533911511062/1037495835690287134/unknown.png
2) https://cdn.discordapp.com/attachments/1021682533911511062/1037495946390540418/unknown.png

File: src/db/special_ability_phase_stat_effects_tables_vco.tsv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+phase	value	stat	how
+#special_ability_phase_stat_effects_tables;3;db/special_ability_phase_stat_effects_tables/vco			
+vco_lord_abilities_youre_next	0.5500	scalar_speed	mult
```
Comment (msolefonte) [inline src/db/special_ability_phase_stat_effects_tables_vco.tsv:3]: alphabetical order

File: src/db/special_ability_phases_tables_vco.tsv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+id	duration	effect_type	requested_stance	unbreakable	cant_move	freeze_fatigue	fatigue_change_ratio	inspiration_aura_range_mod	ability_recharge_change	resurrect	hp_change_frequency	heal_amount	damage_amount	max_damaged_entities	mana_regen_mod	mana_max_depletion_mod	imbue_magical	imbue_ignition	imbue_contact	phase_display	phase_audio	is_hidden_in_ui	affects_allies	affects_enemies	replenish_ammo	composite_scene_group	spreading	freeze_recharge	barrier_heal_amount
+#special_ability_phases_tables;41;db/special_ability_phases_tables/vco																													
+vco_lord_abilities_youre_next	24.0000	negative		false	false	false	0.0000	0.0000	0.0000	false	0.0000	0.0000	0	0	0.0000	0.0000	false	0		wh_main_generic_debuff_defence	wh_ability_negative	false	true	true	0.0000			false	0.0000
```
Comment (msolefonte) [inline src/db/special_ability_phases_tables_vco.tsv:3]: alphabetical order

File: src/db/special_ability_to_special_ability_phase_junctions_tables_vco.tsv [hunk 3]
```diff
@@ -0,0 +1,4 @@
+special_ability	phase	order	target_self	target_friends	target_enemies
+#special_ability_to_special_ability_phase_junctions_tables;2;db/special_ability_to_special_ability_phase_junctions_tables/vco					
+vco_lord_abilities_youre_next	wh2_dlc15_lord_abilities_youre_next	1	false	false	true
```
Comment (msolefonte) [inline src/db/special_ability_to_special_ability_phase_junctions_tables_vco.tsv:3]: alphabetical order

File: src/db/unit_abilities_tables_vco.tsv [hunk 4]
```diff
@@ -0,0 +1,4 @@
+key	superseded_abilities_set	requires_effect_enabling	icon_name	overpower_option	type	video	uniqueness	is_unit_upgrade	is_hidden_in_ui	source_type	is_hidden_in_ui_for_enemy
+#unit_abilities_tables;17;db/unit_abilities_tables/vco											
+vco_lord_abilities_youre_next		true	wh2_dlc15_lord_abilities_youre_next		wh_type_hex		wh_main_anc_group_uncommon	true	false	active	false
```
Comment (msolefonte) [inline src/db/unit_abilities_tables_vco.tsv:3]: alphabetical order

File: src/db/unit_set_unit_ability_junctions_tables_vco.tsv [hunk 5]
```diff
@@ -9,4 +9,4 @@ vco_def_morathi_usa_targeting_range	vco_def_morathi	wh_main_spell_death_soulblig
 vco_def_morathi_usa_targeting_range	vco_def_morathi	wh_dlc05_spell_shadow_melkoths_mystifying_miasma_upgraded
 vco_def_morathi_usa_targeting_range	vco_def_morathi	wh_main_spell_death_doom_and_darkness_upgraded
 vco_def_morathi_usa_targeting_range	vco_def_morathi	wh_dlc05_spell_shadow_pit_of_shades_upgraded
-vco_nor_wulfrik_usa_youre_next	vco_nor_wulfrik	wh2_dlc15_lord_abilities_youre_next
+vco_nor_wulfrik_usa_hunter_of_champions	vco_nor_wulfrik	vco_lord_abilities_hunter_of_champions
```
Comment (msolefonte) [inline src/db/unit_set_unit_ability_junctions_tables_vco.tsv:12]: alphabetical order

File: src/db/unit_special_abilities_tables_vco.tsv [hunk 6]
```diff
@@ -0,0 +1,4 @@
...
+#unit_special_abilities_tables;70;db/unit_special_abilities_tables/vco																																																																				
+vco_lord_abilities_youre_next	-1	24.0000	90.0000	0.0000	0.0000	false	0.0000	false	0	1	false		false	true	false	75.0000	161931632					0.0000	0.0000	wh_abilities_generic_debuff_no_ring				0.0000		hex_melee	wh_group_lord_ability		vo_battle_special_ability_generic_response	150.0000	0.0000				false	false	false	false			false		false	false	false	false				false	false	false	false	true	-1.0000	false	debuff_unit	1.0000	1	false		true
```
Comment (msolefonte) [inline src/db/unit_special_abilities_tables_vco.tsv:3]: alphabetical order

File: src/db/vco_effects.loc.tsv [hunk 7]
```diff
@@ -26,7 +26,7 @@ effects_description_vco_nor_wul_marauder_spear_hunter_bonus_large	Weapon Strengt
 effects_description_vco_nor_wul_marauder_spear_hunter_primal_instincts	Passive ability: "Primal Instincts" for Marauder Spearmen and Hunters	true
 effects_description_vco_nor_wul_movement_range_post_raze_port	Movement Range: %+n% after razing a port settlement	true
 effects_description_vco_nor_wul_units_unlock_dummy	Unlocks new Daemonic units from pledging to a Chaos God	true
-effects_description_vco_nor_wul_ability_enable_youre_next	Ability: "You're Next!"	true
+effects_description_vco_nor_wul_enable_rampage_ability	Ability: "Hunter of Champions"	true
```
Comment (msolefonte) [inline src/db/vco_effects.loc.tsv:29]: alphabetical order

File: src/db/vco_special_ability_phases.loc.tsv [hunk 8]
```diff
@@ -0,0 +1,4 @@
+key	text	tooltip
+#Loc PackedFile;1;text/db/vco_special_ability_phases.loc		
+special_ability_phases_onscreen_name_vco_lord_abilities_youre_next		false
```
Comment (msolefonte) [inline src/db/vco_special_ability_phases.loc.tsv:3]: alphabetical order

File: src/db/effect_bonus_value_ids_unit_sets_tables_vco.tsv [hunk 9]
```diff
@@ -22,4 +22,3 @@ vco_ogr_greasus_ward_save	unit_damage_resistance_all_mod	vco_ogr_greasus
 vco_skv_clan_moulder_strength	melee_damage_mod_mult	wh2_dlc16_clan_moulder_units
 vco_vmp_man_ward_save	unit_damage_resistance_all_mod	vco_vmp_man
 vco_vmp_man_weapon_strength	melee_damage_mod_add	vco_vmp_man
-
```
Comment (msolefonte) [inline src/db/effect_bonus_value_ids_unit_sets_tables_vco.tsv:25]: I just hate that GitHub does this.

File: src/db/special_ability_phase_stat_effects_tables_vco.tsv (hunk 1 above)
Comment (msolefonte) [reply src/db/special_ability_phase_stat_effects_tables_vco.tsv:3]: I am not sure why are we keeping the `vco_lord_abilities_youre_next` ability. Are we using it? I think we are just using `vco_lord_abilities_hunter_of_champions`.

**Review by msolefonte — CHANGES_REQUESTED** [review]
Check comments

File: src/db/special_ability_phase_stat_effects_tables_vco.tsv (hunk 1 above)
Comment (Mjb141) [reply src/db/special_ability_phase_stat_effects_tables_vco.tsv:3]: Correct, removed.

File: src/db/special_ability_phases_tables_vco.tsv (hunk 2 above)
Comment (Mjb141) [reply src/db/special_ability_phases_tables_vco.tsv:3]: Resolved.

File: src/db/unit_set_unit_ability_junctions_tables_vco.tsv (hunk 5 above)
Comment (Mjb141) [reply src/db/unit_set_unit_ability_junctions_tables_vco.tsv:12]: `youre_next` is removed.

File: docs/immortal-empires/norsca.md [hunk 10]
```diff
@@ -25,7 +25,7 @@
 	Skavenblight, Khemri, The Awakening, Wei-Jin, Zhar-Naggrund
   * **Reward**:
 	* _The Eternal Challenger_: 
-	  * Leadership: -20% for enemy chracters near Wulfrik, give him Grimgor or Snitch ability
+	  * Ability: "Hunter of Champions"
```
Comment (msolefonte) [inline docs/immortal-empires/norsca.md:28]: Ability: "Hunter of Champions" (maybe a short description, optional) **for Wulfrik**

File: src/db/effects_tables_vco.tsv [hunk 11]
```diff
@@ -26,7 +26,7 @@ vco_nor_wul_marauder_spear_hunter_bonus_large	monster_hunt.png	101	monster_hunt.
 vco_nor_wul_marauder_spear_hunter_primal_instincts	dlc12_plaque_stage_3.png	101	dlc12_plaque_stage_3.png	campaign	true
 vco_nor_wul_movement_range_post_raze_port	engineer.png	101	engineer.png	campaign	true
 vco_nor_wul_units_unlock_dummy	chaos_gifts.png	101	chaos_gifts.png	campaign	true
```
Comment (msolefonte) [inline src/db/effects_tables_vco.tsv:28]: Still alpha

File: src/db/special_ability_phases_tables_vco.tsv [hunk 12]
```diff
@@ -0,0 +1,4 @@
+id	duration	effect_type	requested_stance	unbreakable	cant_move	freeze_fatigue	fatigue_change_ratio	inspiration_aura_range_mod	ability_recharge_change	resurrect	hp_change_frequency	heal_amount	damage_amount	max_damaged_entities	mana_regen_mod	mana_max_depletion_mod	imbue_magical	imbue_ignition	imbue_contact	phase_display	phase_audio	is_hidden_in_ui	affects_allies	affects_enemies	replenish_ammo	composite_scene_group	spreading	freeze_recharge	barrier_heal_amount
+#special_ability_phases_tables;41;db/special_ability_phases_tables/vco																													
+vco_lord_abilities_hunter_of_champions	28.0000	negative		false	false	false	0.0000	0.0000	0.0000	false	0.0000	0.0000	0	0	0.0000	0.0000	false	0				false	false	true	0.0000			false	0.0000
+vco_lord_abilities_youre_next	24.0000	negative		false	false	false	0.0000	0.0000	0.0000	false	0.0000	0.0000	0	0	0.0000	0.0000	false	0		wh_main_generic_debuff_defence	wh_ability_negative	false	true	true	0.0000			false	0.0000
```
Comment (msolefonte) [inline src/db/special_ability_phases_tables_vco.tsv:4]: This should be out

File: src/db/special_ability_to_invalid_target_flags_tables_vco.tsv [hunk 13]
```diff
@@ -0,0 +1,3 @@
+special_ability	invalid_target
+#special_ability_to_invalid_target_flags_tables;0;db/special_ability_to_invalid_target_flags_tables/vco	
+vco_lord_abilities_youre_next	unit_is_not_commander_class
```
Comment (msolefonte) [inline src/db/special_ability_to_invalid_target_flags_tables_vco.tsv:3]: This should be out/replace with hunter_of_champions.

File: src/db/special_ability_to_special_ability_phase_junctions_tables_vco.tsv [hunk 14]
```diff
@@ -0,0 +1,4 @@
+special_ability	phase	order	target_self	target_friends	target_enemies
+#special_ability_to_special_ability_phase_junctions_tables;2;db/special_ability_to_special_ability_phase_junctions_tables/vco					
+vco_lord_abilities_hunter_of_champions	vco_lord_abilities_hunter_of_champions	1	false	false	true
+vco_lord_abilities_youre_next	wh2_dlc15_lord_abilities_youre_next	1	false	false	true
```
Comment (msolefonte) [inline src/db/special_ability_to_special_ability_phase_junctions_tables_vco.tsv:4]: This should be out

File: src/db/unit_abilities_tables_vco.tsv [hunk 15]
```diff
@@ -0,0 +1,4 @@
+key	superseded_abilities_set	requires_effect_enabling	icon_name	overpower_option	type	video	uniqueness	is_unit_upgrade	is_hidden_in_ui	source_type	is_hidden_in_ui_for_enemy
+#unit_abilities_tables;17;db/unit_abilities_tables/vco											
+vco_lord_abilities_hunter_of_champions		true	wh_dlc08_lord_abilities_fight_or_die		wh_type_hex		wh_main_anc_group_uncommon	true	false	active	false
+vco_lord_abilities_youre_next		true	wh2_dlc15_lord_abilities_youre_next		wh_type_hex		wh_main_anc_group_uncommon	true	false	active	false
```
Comment (msolefonte) [inline src/db/unit_abilities_tables_vco.tsv:4]: This should be out

File: src/db/unit_special_abilities_tables_vco.tsv [hunk 16]
```diff
@@ -0,0 +1,4 @@
...
+#unit_special_abilities_tables;70;db/unit_special_abilities_tables/vco																																																																				
+vco_lord_abilities_hunter_of_champions	1	50.0000	90.0000	0.0000	0.0000	false	0.0000	false	0	1	false		false	true	false	100.0000	1367343187					0.0000	0.0000	wh_abilities_generic_debuff_no_ring				0.0000		hex_melee	wh_group_lord_ability		vo_battle_special_ability_generic_response	150.0000	0.0000				true	false	false	false			false		false	false	false	false				false	false	false	false	true	-1.0000	false	debuff_unit	1.0000	3	false		true		
+vco_lord_abilities_youre_next	-1	24.0000	90.0000	0.0000	0.0000	false	0.0000	false	0	1	false		false	true	false	75.0000	161931632					0.0000	0.0000	wh_abilities_generic_debuff_no_ring				0.0000		hex_melee	wh_group_lord_ability		vo_battle_special_ability_generic_response	150.0000	0.0000				false	false	false	false			false		false	false	false	false				false	false	false	false	true	-1.0000	false	debuff_unit	1.0000	1	false		true
```
Comment (msolefonte) [inline src/db/unit_special_abilities_tables_vco.tsv:4]: This should be out

File: src/db/vco_effects.loc.tsv [hunk 17]
```diff
@@ -38,3 +38,4 @@ effects_description_vco_skv_per_turn_mutagen	Mutagen: %+n per turn	true
 effects_description_vco_tmb_arkhan_allegiance_points_mod_vampire_coast	Allegiance points gained: %+n% for alliances with Vampire Coast	true
 effects_description_vco_vmp_man_ward_save	Ward save: %+n% for Mannfred von Carstein	true
 effects_description_vco_vmp_man_weapon_strength	Weapon strength: %+n% for Mannfred von Carstein	true
+effects_description_vco_effect_ability_enable_youre_next	Ability: "You're Next!"	false
```
Comment (msolefonte) [inline src/db/vco_effects.loc.tsv:41]: This should be out

File: src/db/vco_special_ability_phases.loc.tsv (hunk 8 above)
Comment (msolefonte) [inline src/db/vco_special_ability_phases.loc.tsv:3]: This should be out

File: src/db/effects_tables_vco.tsv [hunk 18]
```diff
@@ -26,7 +26,7 @@ vco_nor_wul_marauder_spear_hunter_bonus_large	monster_hunt.png	101	monster_hunt.
...
-vco_nor_wul_ability_enable_youre_next	wh2_dlc15_lord_abilities_youre_next.png	1	wh2_dlc15_lord_abilities_youre_next.png	campaign	true
+vco_nor_wul_enable_rampage_ability	wh2_dlc15_lord_abilities_youre_next.png	1	wh2_dlc15_lord_abilities_youre_next.png	campaign	true
```
Comment (msolefonte) [inline src/db/effects_tables_vco.tsv:29]: Maybe naming it Hunter of Champions directly? This sounds like standard rampage.

File: src/db/vco_effects.loc.tsv [hunk 19]
```diff
@@ -21,12 +21,12 @@ effects_description_vco_grn_black_orcs_frostbite	Enables frostbite attacks for B
 effects_description_vco_grn_per_turn_salvage_gain	Scrap: %+n per turn	true
 effects_description_vco_grn_per_turn_waaagh	Reputation: %+n per turn	true
 effects_description_vco_ksl_tgo_luminark_unlock_dummy	Unlock new unit: Luminark of Hysh	true
+effects_description_vco_nor_wul_enable_rampage_ability	Ability: "Hunter of Champions"	true
```
Comment (msolefonte) [inline src/db/vco_effects.loc.tsv:24]: Maybe naming it Hunter of Champions directly?

**Review by msolefonte — CHANGES_REQUESTED** [review]

**Review by msolefonte — APPROVED** [review]

**Pull request merged**